File: Maths/degRad.js
/**
 * @description Fonction qui permet de passer de degrés en radians.
 * @param {number} deg - Degrés
 * @returns {number} Radians
 */
const degEnRad = (deg) => (deg * Math.PI) / 180;

/**
 * @description Fonction qui permet de passer de radians en degrés.
 * @param {number} rad - Radians
 * @returns {number} Degrés
 */
const radEnDeg = (rad) => (rad * 180) / Math.PI;

// Tests
console.table({
  "degEnRad(90)": degEnRad(90),
  "radEnDeg(π / 4)": radEnDeg(Math.PI / 4),
});

File: Python/Maths/deg_rad.py
from math import pi

def deg_en_rad(deg: float) -> float:
    """
    Description:
        Fonction qui permet de passer de degrés en radians.
    
    Paramètres:
        deg: {float} -- Degrés à convertir.
    
    Retourne:
        {float} -- Radians.
    
    Exemple:
        >>> degEnRad(90)
        1.5707963267948966
    """
    return deg * (pi / 180)

def rad_en_Deg(rad: float) -> float:
    """
    Description:
        Fonction qui permet de passer de radians en degrés.
    
    Paramètres:
        rad: {float} -- Radians à convertir.
    
    Retourne:
        {float} -- Degrés.
    
    Exemple:
        >>> radEnDeg(1.5707963267948966)
        90.0
    """
    return rad * (180 / pi)